### ISSUE TITLE
exif data is lost while exporting to jpg

### ISSUE THREAD

**Issue by markushuebner** [darktable-org/darktable#7524]
**Describe the bug**

Exported jpg from Canon CR2 does not contain any of the exif data within the raw file.

**To Reproduce**

1. Choose image within lighttable, exif data of cr2 file is shown on the left hand side
2. edit the image within darkroom
3. turn back to lighttable and set export format to 8bit jpg
4. check the cog right next to the export-button to ensure, that the checkbox for exif data in general settings is set, nothing else is activated
5. export
6. open exported jpg with any kind of viewer (e.g. faststone)
7. check exif data - which is empty

**Expected behavior**

The exported jpg should contain the original exif data of the raw file.

**Screenshots**

![image](https://user-images.githubusercontent.com/47855130/103243283-a68b5c00-4959-11eb-930f-3e18f81d3b64.png)
![image](https://user-images.githubusercontent.com/47855130/103243243-8cea1480-4959-11eb-86c8-012b6aa920dd.png)
![image](https://user-images.githubusercontent.com/47855130/103243337-d470a080-4959-11eb-91bd-a4c86f6170de.png)

**Platform (please complete the following information):**
 - Darktable Version: 3.2.1
 - OS: Windows 10 ARM, 64bit emulated
 - OpenCL is not activated
 - CPU: SD835, GPU: Adreno 540
 - model of computer: Lenovo Miix630 

**Additional context**

**Comment by phweyland** [comment]
I cannot reproduce this with current master and NEF files.
Could you share the output of:
`exiv2 -PE <yourfile.jpg>`
Thanks

**Comment by markushuebner** [comment]
Sorry for this question, but - how do I create 
_exiv2 -PE <yourfile.jpg>_
?

**Comment by phweyland** [comment]
Sorry I don't know if exiv2 is available on windows. Exiftool does exist and is a command line utility, but I don't remember the exact syntax.
Simpler way... could you share such a faulty jpg file ?

**Comment by markushuebner** [comment]
This is strange. I imported a cr2-file from another folder without any people on it to create a test-jpg and upload it here - and all exif-data is still contained in the exported jpg.
Then I re-imported the pictures which suffered from losing exif data and exported one of them again. - Result: Exif data is contained within the created jpg as it should be.
So, it seems to me that I accidentally clicked something within here and refreshing exif means "strip it all" on every image in the imported folder? 
![image](https://user-images.githubusercontent.com/47855130/103308152-0c3b1f00-4a12-11eb-9d6b-ac2f55b413ca.png)
I kept the cr2-file marked and clicked it to repeat the export. As expected, no exif-data is found within the created jpg.
The I chose another cr2-file, which was not marked and exported it - exif-data found its way into the jpg.

I can't explain the issues I had before, besides the possible reason that I accidentally had all my images selected and then I clicked at "refresh exif" so all images lost their data for exporting them. Combined with not changing the folders since then it let to losing the data on every export - despite closing and opening darktable several times and restarting my computer. Importing the files again might have brought the exif-data back.
